### DISCUSSION TITLE
GH Copilot is not showing suggestions in VSCode

### DISCUSSION THREAD

**Discussion by RicardoNiepel** [community/community#108500] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

As you can see in the screenshot, GHCopilot is unable to show suggestions in VSCode with anything up-to-date:
![image](https://github.com/community/community/assets/7172277/f7abfc8e-1dc9-4844-8b07-31e653970ece)

# Troubleshooting info
request done: headerRequestId: [c4293a68-a1e0-4aed-bbb0-d4010777fce6] model deployment ID: [y5aa6d170bcae]

## Copilot

- Version: 1.162.0
- Build: prod
- Editor: vscode/1.86.2

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Custom Certificates: disabled
- Send Restricted Telemetry: enabled
- Chat: enabled
- Chat JetBrains: enabled

## Node setup

- Number of root certificates: 137
- Operating system: Windows_NT
- Operating system version: 10.0.22621
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Network Configuration

- Proxy host: n/a
- Proxy port: n/a
- Kerberos SPN: n/a
- Reject unauthorized: disabled
- Fetcher: HelixFetcher

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- api.githubcopilot.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 

## VS Code Configuration

- HTTP proxy: 
- HTTP proxy authentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Extensions

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false

## Authentication

- GitHub username: RicardoNiepel

**Comment by gdomingu** [comment]
Thank you for the report. From the screenshot it looks like there is no ghost text appearing. I wonder if the logs showing `solution x returned` might be a red herring and in fact there are no suggestions? Does it show suggestions if you open the panel when there is ghost text?

**Comment by davecheney** [comment, marked as answer]
@RicardoNiepel the response was redacted because it matched public code. If you wish you can disable the "block public suggestions" feature in https://github.com/settings/copilot

**Comment by RicardoNiepel** [comment]
@davecheney How is it possible to see as normal user what it was redacted because of this reason or maybe for another reason (PII / violent language)?

**Reply by davecheney** [reply]
Sadly we do not expose that information to end users.
